gps-rtk-serial-no-network/gps_rtk_serial_no_network_test.go

```diff
@@ -0,0 +1,237 @@
+package gpsrtkserialnonetwork
+
+import (
+	"context"
+	"io"
+	"strings"
+	"testing"
+
+	"github.com/edaniels/golog"
+	"github.com/golang/geo/r3"
+	geo "github.com/kellydunn/golang-geo"
+	"go.viam.com/rdk/components/movementsensor"
+	"go.viam.com/rdk/components/movementsensor/gpsnmea"
+	"go.viam.com/rdk/resource"
+	"go.viam.com/test"
+	"go.viam.com/utils"
+)
+
+const nmeaPath = "nmea-path"
+const correctionPath = "corr-path"
+
+var mockGPSData = gpsnmea.GPSData{
+	Location:   geo.NewPoint(1, 2),
+	Alt:        3,
+	Speed:      4,
+	VDOP:       5,
+	HDOP:       6,
+	SatsInView: 7,
+	SatsInUse:  8,
```

Comment:
nit - testing bad data here would be nice to see how the functions handle it.

Reply:
I have tests for when the location is 0,0 and nil. Are there any other test cases you think should be added?

Reply:
floating points that's really close to zero would be a good test case. Also checking if last position updates.